backend/capellacollab/projects/exceptions.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from fastapi import status
+
+from capellacollab.core import exceptions as core_exceptions
+
+from . import models
+
+
+class ProjectNotFoundError(core_exceptions.BaseError):
+    def __init__(self, project_slug: str):
+        super().__init__(
+            status_code=status.HTTP_404_NOT_FOUND,
+            title="Project not found",
+            reason=f"The project with the slug '{project_slug}' was not found.",
+            err_code="PROJECT_NOT_FOUND",
+        )
+
+
+class ProjectAlreadyExistsError(core_exceptions.BaseError):
+    def __init__(self, project_slug: str):
+        super().__init__(
+            status_code=status.HTTP_409_CONFLICT,
+            title="Project already exists",
+            reason=f"The project with the slug '{project_slug}' (or similar) already exists.",
```

Comment:
```suggestion
            reason=f"The project with the slug '{project_slug}' already exists.",
```
I think the "or similar" is more confusing then it actually helps here, so I would just remove it.